.github/workflows/automerge.yml

```diff
@@ -62,3 +62,11 @@ jobs:
             additional_check_runs = String["CI (1, ubuntu-latest, x64)",],
           )
         shell: julia --color=yes --project=.ci/ {0}
+  AutoMerge-stopwatch:
+    if: github.event_name == 'pull_request'
+    environment: stopwatch
+    runs-on: ubuntu-latest
+    # needs: AutoMerge # TODO: uncomment this line
+    steps:
+      # TODO: this step should run the "cron" code path
```

Comment:
If the cron code path requires secrets, you may have to switch to the `pull_request_target` event and pull in the PR files manually in the first job, or separate the jobs into different workflows (and add a check to the merging step that the check run has actually finished).

Reply:
We only run AutoMerge on PRs made by JuliaRegistrator, which has write access to this repo. So the `pull_request` event should have access to secrets, right? Since the PRs are not made from forks.

Reply:
Ah, yes. I didn't know about that, I thought it ran on all PRs.